pom.xml

```diff
@@ -31,9 +31,24 @@
         <spock.version>1.1-groovy-2.4</spock.version>
         <cglib.version>3.2.5</cglib.version>
         <groovy.version>2.4.15</groovy.version>
+        <snakeyaml.version>1.21</snakeyaml.version>
+        <lombok.version>1.18.2</lombok.version>
+        <commons.version></commons.version>
     </properties>
 
     <dependencies>
+        <!-- YAML Reading -->
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>${snakeyaml.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
+            <scope>provided</scope>
+        </dependency>
         <!-- Tests -->
```

Comment:
What about one empty line separator to keep better readability of the dependencies groups?